Moli/Core/Components/SnapCarousel/SnapCarouselItem.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+
+struct SnapCarouselItem<Content: View>: View {
+    @EnvironmentObject var UIState: SnapCarouselViewModel
+    let cardWidth: CGFloat
+    let cardHeight: CGFloat
+    
+    var _id: Int
+    var content: Content
+    
+    @inlinable public init(
+        _id: Int,
+        spacing: CGFloat,
+        widthOfHiddenCards: CGFloat,
+        cardHeight: CGFloat,
+        @ViewBuilder _ content: () -> Content
+    ) {
+        self.content = content()
+        self.cardWidth = UIScreen.main.bounds.width - (widthOfHiddenCards*2) - (spacing*2)
+        self.cardHeight = cardHeight
+        self._id = _id
+    }
+    
+    var body: some View {
+        content
+            .frame(width: cardWidth, height: _id == UIState.activeCard ? cardHeight : cardHeight - 20, alignment: .center)
```

Comment:
❓ Skulle vi köra utan scale effekten?
```suggestion
            .frame(width: cardWidth, height: cardHeight, alignment: .center)
```